examples/winrm/windows_services.js

```diff
@@ -0,0 +1,142 @@
+/**
+ * Domotz Custom Driver 
+ * Name: Windows Services Monitoring
+ * Description: Monitors the status of services on a Windows machine
+ *   
+ * Communication protocol is WinRM
+ * 
+ * Tested on Windows Versions:
+ *      - Windows 10
+ *      - Microsoft Windows Server 2019
+ * Powershell Version:
+ *      - 5.1.19041.2364
+ * 
+ * Creates a Custom Driver table with a list of services, their status and their start type
+ * 
+ * Privilege required: Local Administrator
+* 
+**/
+
+// List of services you want to monitor, note that you can put the DisplayName or the ServiceName  
+
+// This is an example of the filter which suit a Windows 10 computer (workstation):
+// var svcFilter = '@("bits","Dnscache","Spooler","schedule","DHCP Client")'
+ 
+// For a server you may want to set the filter as follow :
+var svcFilter = '@("LanmanServer","dnscache","Windows Time","dhcp","schedule","RpcEptMapper","MpsSvc")'
```

Comment:
It will be easier to use if this filter variable is a list/array

Below you can use '@(' + svcFilter.toString() + ')'